### PR TITLE
Create README.ru.md

### PR DESCRIPTION
**Pull request by ViktorOn**
Changes:

- Added translation to Russian language of the file README

### CODE REVIEW

File: doc/README.ru.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+* Обзор и поиск истории просмотра
+* Поиск и просмотр списков воспроизведения (это удаленные списки воспроизведения, что означает, что они получены из сервиса, которую вы просматриваете)
+* Создавайте и редактируйте локальные списки воспроизведения (они создаются и сохраняются в приложении и не имеют ничего общего с каким-либо сервисом)
+* Скачивайте видео/аудио/субтитры (скрытые титры)
+* Откройте в Kodi
+* Смотрите/блокируйте материалы с возрастным ограничением
+
+<!-- Hidden span to keep old links compatible. -->
+<span id="updates"></span>
+
```
Comment (mhmdanas) [inline doc/README.ru.md:91]: There's no point in having this if the README is brand new:
```suggestion
```

File: doc/README.ru.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+<a href="https://www.bountysource.com/teams/newpipe" alt="Награды Bountysource"><img src="https://img.shields.io/bountysource/team/newpipe/activity.svg?colorB=cd201f"></a>
+</p>
+<hr>
+<p align="center"><a href="#screenshots">Скриншоты</a> &bull; <a href="#supported-services">Поддерживаемые сервисы</a> &bull; <a href="#description">Описание</a> &bull; <a href="#features">Возможности</a> &bull; <a href="#installation-and-updates">Установка и обновления</a> &bull; <a href="#contribution">Участие</a> &bull; <a href="#donate">Пожертвование</a> &bull; <a href="#license">Лицензия</a></p>
+<p align="center"><a href="https://newpipe.net">Сайт</a> &bull; <a href="https://newpipe.net/blog/">Блог</a> &bull; <a href="https://newpipe.net/FAQ/">ЧЗВ</a> &bull; <a href="https://newpipe.net/press/">Пресса</a></p>
+<hr>
+
+*Эта страница на других языках: [English](README.md), [Español](doc/README.es.md), [हिन्दी](doc/README.hi.md), [한국어](doc/README.ko.md), [Soomaali](doc/README.so.md), [Português Brasil](doc/README.pt_BR.md), [Polski](doc/README.pl.md), [日本語](doc/README.ja.md), [Română](doc/README.ro.md), [Türkçe](doc/README.tr.md), [正體中文](doc/README.zh_TW.md).*
```
Comment (mhmdanas) [inline doc/README.ru.md:20]: Please add a link to the Russian README to the end of this line, here and in all the other READMEs.

**Review by mhmdanas — CHANGES_REQUESTED** [review]
Please add a link to the Russian README to all the READMEs, _

**Review by mhmdanas — COMMENTED** [review]
I went ahead resolved my review myself. LGTM now.

**Comment by ViktorOn** [comment]
@mhmdanas Oh, that's great news. I've been busy, so it's good that you solved this issue.

**Comment by TobiGr** [comment]
Thank you for your work. Your changes were merged in #10240

**Pull request closed without merging**